cypress/support/pages/course/pageCourses.js

```diff
@@ -11,6 +11,8 @@ class Courses {
   static #courseDescription = '[id="courseDescription"]'
   static #courseName = '[name="name"]'
   static #createFAB = '[name="fab-icon"]'
+  static #seachFieldRoomOverview = '[data-testid="search-field"]'
```

Comment:
Typo: seachFieldRoomOverview -> searchFieldRoomOverview (missing the r)

Reply:
corrected the typo.